config/state-machine.php

```diff
@@ -15,8 +15,9 @@
             'printed',
             'dispatched',
             'allocated', // to families
-            'recorded',
-            'reimbursed',
+            'recorded', // submitted to shortlist
+            'payment_pending', // shortlist completed
+            'reimbursed', // paid out
```

Comment:
If you've got any suggestions for renaming states and the transitions to get to them there's still time to change it to be more context-y!